app/dashboard/models.py

```diff
@@ -357,10 +370,10 @@ def status(self):
         else:
             try:
                 if not self.is_open:
-                    if timezone.localtime().replace(tzinfo=None) > self.expires_date.replace(tzinfo=None) and self.num_fulfillments == 0:
-                        return 'expired'
                     if self.accepted:
                         return 'done'
+                    if self.past_hard_expiration_date:
```

Comment:
Moving this if check above `if self.accepted:` should resolve test failures, I'm thinking 🤔